client/components/domains/domain-suggestion/style.scss

```diff
@@ -44,26 +44,9 @@
 
 	.notice.is-compact {
 		margin: 0 0 0 8px;
-		color: $white;
-
-		&.is-success {
-			background: $alert-green;
-		}
-
-		&.is-info {
-			background: $blue-wordpress;
-		}
-
-		&.is-warning {
-			background: $alert-yellow;
-		}
-
-		.notice__content {
-			padding: 4px 8px;
-		}
```

Comment:
Would it be possible to explain why we need the notice style changes?

Reply:
The only thing I changed in this file was adding that ", &:focus" on line 19. Possibly git is confused?

Reply:
Bad rebase, maybe?